src/compiler/diagnosticMessages.json

```diff
@@ -3938,8 +3938,12 @@
         "category": "Message",
         "code": 95016
     },
+    "Mapped object type implicitly has an 'any' template type.": {
+        "category": "Error",
+        "code": 95017

```

Comment:
This should be closer to the other semantic diagnostics numbers (right before the 8000 range)